packages/cli-tools/src/doclink.ts

```diff
@@ -62,7 +62,10 @@ function doclink(
     : `${section}/${path}`;
 
   url.searchParams.set('os', OS);
-  url.searchParams.set('platform', platform);
+
+  if (platform) {
+    url.searchParams.set('platform', platform);
+  }
```

Comment:
I don't have context on this code, but is there a way we can force relevant call sites to specify a platform where one is needed?

Reply:
I'm not sure I understand the question, but giving it a go.  In some places where we take care of this.  For example when the user runs `npx react-native run-android`, we set the [platform to android](https://github.com/react-native-community/cli/blob/main/packages/cli-platform-android/src/commands/runAndroid/index.ts#L41).

Is that what you're asking about?

Reply:
What I mean is: Is it useful that `platform` is optional on line 31? Could we, for example, make it required in every call to `doclink`?